src/Sentry/Protocol/Transaction.cs

```diff
@@ -56,6 +56,8 @@ public void Finish(SpanStatus status = SpanStatus.Ok)
         {
             EndTimestamp = DateTimeOffset.Now;
             Status = status;
+
+            SentrySdk.CaptureTransaction(this);
```

Comment:
we wont' be able to unit test this unless we take `IHub` here.

Hub could pass `this` when creating the instance

Reply:
Yeah, this is a hacky implementation.
Overall, I really dislike sending stuff from `Transaction.Finish`

Reply:
Same here. At firs tin Java we didn't do that but rather you should call Hub.capture.